libraries/AP_RangeFinder/AP_RangeFinder_DFRobot_Lidar07.cpp

```diff
@@ -0,0 +1,241 @@
+/*
+   This program is free software: you can redistribute it and/or modify
+   it under the terms of the GNU General Public License as published by
+   the Free Software Foundation, either version 3 of the License, or
+   (at your option) any later version.
+
+   This program is distributed in the hope that it will be useful,
+   but WITHOUT ANY WARRANTY; without even the implied warranty of
+   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+   GNU General Public License for more details.
+
+   You should have received a copy of the GNU General Public License
+   along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#include "AP_RangeFinder_DFRobot_Lidar07.h"
+
+#if AP_RANGEFINDER_DFROBOT_LIDAR07_ENABLED
```

Comment:
```suggestion
#include "AP_RangeFinder_config.h"

#if AP_RANGEFINDER_DFROBOT_LIDAR07_ENABLED

#include "AP_RangeFinder_DFRobot_Lidar07.h"
```

... I know this isn't what all of the other headers in here do, but this is the pattern we're trying to use in ArduPilot now where config files are involved.

Reply:
@peterbarker got it. I took the NMEA as a template, will change.